ci-operator/step-registry/openshift/e2e/vsphere/upi/release46/openshift-e2e-vsphere-upi-release46-workflow.yaml

```diff
@@ -0,0 +1,15 @@
+workflow:
+  as: openshift-e2e-vsphere-upi-release46
```

Comment:
This workflow is incredibly useful for OKD jobs, which uses a lot of docker.io images. Lets rename this to `openshift-e2e-vsphere-upi-pullthrough` or similar?

Reply:
The reason for specifically calling out release-4.6 in the name is due to the fact that this configuration is not necessary in 4.7 and later as the targeted tests in openshift-tests already pull from other sources.  The bulk of the tests which failed due to rate limiting are fixed by configuring a mirror in crio.

Reply:
Ah, I see, okay